R/pkg/R/DataFrame.R

```diff
@@ -1859,7 +1859,7 @@ setMethod("where",
 #' @param colnames A character vector of column names.
```

Comment:
Oh, thank you for review, @shivaram .
Sure. I'll update the doc. Maybe, something like the following?
```
- #' @param colnames A character vector of column names.
+ #' @param col A character vector of column names or a string of column name
+ #' @param ... Additional column names
```